core/src/main/java/org/mskcc/cbio/portal/servlet/QueryBuilder.java

```diff
@@ -330,7 +330,8 @@ protected void doPost(HttpServletRequest httpServletRequest,
     private void processData(CohortDetails cohortDetails,
                              String geneList,
 							 HashSet<String> geneticProfileIdSet,
-							 String sampleSetId, String sampleIds,
+							 String sampleSetId, 
+                             String sampleIdsStr, //raw string from "user custom case ids" box
```

Comment:
It would be the same. My thinking was to have process data to accept all raw parameters from the query form or summary page query. And there's also dispatch these parameters back to the result page, so I thought it would be confusing to process it outside "processData". Please let me know what you think.